lib/responses.js

```diff
@@ -146,6 +146,8 @@ function notImplemented(req, res) {
   const code = 501,
     message = 'Not Implemented';
 
+  log('error', message, { code });
+
```

Comment:
This is going to produce a log line like this:
```
{ "level": 50, "message": "Not Implemented", "code": 501 }
```

I think in general if we are in this block there was a deliberate choice _before_ this block to put us here (ie. setting `err.status = 501`).
Whatever context for _why_ we got here is only going to exist in that earlier location.

Reply:
The only areas this function is called is in `_lists` and `_uris` logic for posting to '/' route. err.status is not manually set before this.